packages/synapse-react-client/src/mocks/msw/handlers/schemaHandlers.ts

```diff
@@ -0,0 +1,53 @@
+// MSW handlers for JSON Schema services
+import {
+  mockConditionalOnConcreteTypeSchema,
+  mockFileEntityValidationSchema,
+  mockProjectValidationSchema,
+  mockValidationSchema,
+} from '@/mocks/mockSchema'
+import { generateAsyncJobHandlers } from '@/mocks/msw/handlers/asyncJobHandlers'
+import {
+  SCHEMA_VALIDATION_GET,
+  SCHEMA_VALIDATION_START,
+} from '@/utils/APIConstants'
+import { JSONSchema7 } from 'json-schema'
+
+const validationSchemas: JSONSchema7[] = [
+  mockValidationSchema,
+  mockConditionalOnConcreteTypeSchema,
+  mockFileEntityValidationSchema,
+  mockProjectValidationSchema,
+]
+
+export function getValidationSchemaHandlers(backendOrigin?: string) {
```

Comment:
Add a basic handler so we can register more than one schema. This is a required change as our component now gets 2 validation schemas -- one for the bound schema and one for the base entity schema